src/storage/src/table/cell_based_table.rs

```diff
@@ -190,40 +200,41 @@ impl<S: StateStore, SER: RowSerializer> From<CellBasedTableBase<S, SER, READ_WRI
 }
 
 impl<S: StateStore, SER: RowSerializer, const T: AccessType> CellBasedTableBase<S, SER, T> {
+    pub(super) fn fallback_vnodes() -> Arc<Bitmap> {
```

Comment:
So this function should only be used by singleton executor in future?

Reply:
Yep. Besides, if we don't want to enable consistent hash for some executors, we can also use this fallback distribution to always encode and decode `0x00` in the key.